.container/Dockerfile

```diff
@@ -0,0 +1,10 @@
+FROM golang:1.23-bookworm
```

Comment:
Add two stage build, like [here](https://medium.com/@kittipat_1413/optimizing-multi-stage-builds-with-dockerfile-in-golang-a2ee8ed37ec6)?

Reply:
Nvm, maybe in next steps if we don't want to push changes to the upstream.